src/main/java/WayofTime/alchemicalWizardry/common/summoning/meteor/Meteor.java

```diff
@@ -21,6 +21,8 @@ public class Meteor {
     public String focusModId;
     public String focusName;
     public int focusMeta;
+    private String[] filler;
+    private int fillerChance;
```

Comment:
Either make both of these `public` too, or (better) make anything that is not needed outside of the class also `private`.